core/postfix/start.py

```diff
@@ -36,7 +36,9 @@ def start_podop():
 
 if os.path.exists("/overrides/postfix.cf"):
     for line in open("/overrides/postfix.cf").read().strip().split("\n"):
-        os.system('postconf -e "{}"'.format(line))
+        line = line.strip()
+        if line and line[0] != "#":
+            os.system('postconf -e "{}"'.format(line))
```

Comment:
What is the puprose of this change? it is not described in the PR title or body? This should go in a separate PR.